Library/Formula/fpc.rb

```diff
@@ -28,11 +28,16 @@ def install
 
     # Prevent non-executable audit warning
     rm_f Dir[bin/"*.rsj"]
+
+    # Generate a default fpc.cfg to facilitate search paths
+    fpcversion = `#{bin}/fpc -iW`.chomp
+    system "#{bin}/fpcmkcfg", "-p", "-d", "basepath=#{lib}/fpc/#{fpcversion}", "-o", "#{prefix}/etc/fpc.cfg"
```

Comment:
Probably we can just use `#{version}` instead of `#{fpcversion}` as they should match.